Source/WebCore/platform/LengthPoint.h

```diff
@@ -32,31 +32,12 @@ namespace WebCore {
 struct BlendingContext;
 
 struct LengthPoint {
-public:
-    LengthPoint()
-    {
-    }
-    
-    LengthPoint(Length x, Length y)
-        : m_x(WTFMove(x))
-        , m_y(WTFMove(y))
-    {
-    }
-
-    friend bool operator==(const LengthPoint&, const LengthPoint&) = default;
-
-    void setX(Length x) { m_x = WTFMove(x); }
-    const Length& x() const { return m_x; }
-
-    void setY(Length y) { m_y = WTFMove(y); }
-    const Length& y() const { return m_y; }
-
-    bool isZero() const { return m_x.isZero() && m_y.isZero(); }
-
-private:
     // FIXME: it would be nice to pack the two Lengths together better somehow (to avoid padding between them).
     Length m_x;
```

Comment:
Our coding conventions suggest that structure members don't have the `m_` prefix

Reply:
Understood. I'll fix that.